packages/utilities-react/src/StepFlow/types.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Dispatch, SetStateAction } from 'react';
+
+export type StepContextType = StepState;
```

Comment:
Should `StepState` be renamed to `StepContextType`? `StepState` is only used in this file.